config/locales/es.yml

```diff
@@ -201,7 +201,12 @@ es:
         street_address: Dirección
         zip: Código postal```
```

Comment:
Double checking the apostrophes after "postal" are intentional. If not, please remove.

Reply:
they were not! Thanks for spotting!